.github/CODEOWNERS

```diff
@@ -1318,6 +1318,10 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 #CC# /src/plugins/kibana_react/public/code_editor/ @elastic/kibana-presentation
 
 # Machine Learning
+/x-pack/test/functional_with_es_ssl/apps/discover_ml_uptime/config.ts @elastic/ml-ui
+/x-pack/test/stack_functional_integration/apps/ml @elastic/ml-ui
+/x-pack/test/functional/fixtures/kbn_archiver/ml @elastic/ml-ui
+/x-pack/test/accessibility/apps/group2/ml_* @elastic/ml-ui
 /x-pack/test/api_integration/apis/file_upload @elastic/ml-ui
 /x-pack/test/accessibility/apps/group2/ml.ts  @elastic/ml-ui
```

Comment:
Could you please group this file with the `/x-pack/test/accessibility/apps/group2/ml.ts` entry a few lines down.